Makefile

```diff
@@ -51,18 +51,26 @@ _output/share/lima/lima-guestagent.Linux-aarch64:
 	chmod 644 $@
 
 .PHONY: install
+install: | $(abspath $(DESTDIR)/$(PREFIX))
+install: DEST = $(realpath $(DESTDIR)/$(PREFIX))
 install:
-	cp -av _output/* "$(DESTDIR)/$(PREFIX)/"
-	if [[ $(shell uname -s ) != Linux && ! -e "$(DESTDIR)/$(PREFIX)/bin/nerdctl" ]]; then ln -sf nerdctl.lima "$(DESTDIR)/$(PREFIX)/bin/nerdctl"; fi
+	@test -d "$(DEST)" || (echo "invalid destination: $(DESTDIR)/$(PREFIX)"; exit 1)
+	cp -av _output/* "$(DEST)/"
+	if [[ $(shell uname -s ) != Linux && ! -e "$(DEST)/bin/nerdctl" ]]; then ln -sf nerdctl.lima "$(DEST)/bin/nerdctl"; fi
 
 .PHONY: uninstall
+uninstall: DEST = $(realpath $(DESTDIR)/$(PREFIX))
 uninstall:
+	@test -d "$(DEST)" || (echo "invalid destination: $(DESTDIR)/$(PREFIX)"; exit 1)
```

Comment:
I would maybe check for `-f "${DEST}/bin/lima` instead, to verify not just that the directory exists, but also that lima has previously been installed into this location.